### PR TITLE
findutils 4.8.0

### PR DESCRIPTION
**Pull request by carlocab**
Follow up to #68646.

-----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by carlocab** [comment]
License header ref (`gl/lib/fstat.c`):
```
/* fstat() replacement.
   Copyright (C) 2011-2021 Free Software Foundation, Inc.

   This program is free software: you can redistribute it and/or modify
   it under the terms of the GNU General Public License as published by
   the Free Software Foundation; either version 3 of the License, or
   (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <https://www.gnu.org/licenses/>.  */
```

**Comment by carlocab** [comment]
Different set of errors this time:
```
Undefined symbols for architecture x86_64:
  "_re_compile_pattern", referenced from:
      _insert_regex in libfindtools.a(parser.o)
  "_re_match", referenced from:
      _pred_regex in libfindtools.a(pred.o)
  "_re_set_syntax", referenced from:
      _insert_regex in libfindtools.a(parser.o)
ld: symbol(s) not found for architecture x86_64
```

**Comment by carlocab** [comment]
Reported upstream. This should appear at https://lists.gnu.org/archive/html/bug-findutils/2021-01/index.html.

**Comment by Diatrus** [comment]
FYI adding `CFLAGS="$(CFLAGS) -D__nonnull\(params\)="` to the end of configure (or the home-brew equivalent of that) will result in a successful compile with clang and without `--without-included-regex`.

**Comment by carlocab** [comment]
Thanks for the tip, @diatrus. What exactly does that flag do?

**Comment by Diatrus** [comment]
@carlocab It defines the `__nullable` macro differently (how the glibc code wants it). You need to make sure *in* the CFLAGS the `\` are present. I can see from the build log they are not. Try escaping the `\`.

**Comment by carlocab** [comment]
>It will also only work if you get rid of the without-included-regex flag you added in.

Whoops, missed that one. Thanks for the reminder.

File: Formula/findutils.rb [hunk 1]
```diff
@@ -29,11 +29,19 @@ def install
...
+    ENV.append "CFLAGS", "-D__nonnull\\(params\\)="
+
     args = %W[
       --prefix=#{prefix}
       --localstatedir=#{var}/locate
       --disable-dependency-tracking
       --disable-debug
+      --disable-nls
+      --with-packager=Homebrew
+      --with-packager-bug-reports=https://github.com/Homebrew/homebrew-core/issues
```
Comment (SMillerDev) [inline Formula/findutils.rb:44]:
```suggestion
      --with-packager-bug-reports=#{tap.issues_url}
```

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**